### PR TITLE
Add 4.1 rules for deployment image

### PR DESCRIPTION
**Pull request by xiaocwan**
@hasha @yanpzhan Plz review, thanks!

### CODE REVIEW

File: lib/rules/web/admin_console/4.1/deployment.xyaml [hunk 1]
```diff
@@ -27,4 +32,19 @@ search_image:
...
+search_nonexisted_image_and_check_message:
+  params:
+    search_content: non-existed
+    content: Could not load image metadata
+  action: search_image
+  action: check_page_contains
+search_and_deploy_image:
+  action: search_image
+  action: wait_form_loaded
+  action: click_deploy_button
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.1/deployment.xyaml]: click_deploy_button could be replaced by click_button, pass button_text parameter.

Comment (xiaocwan) [reply lib/rules/web/admin_console/4.1/deployment.xyaml]: Cool, updated.

**Comment by yanpzhan** [comment]
@xiaocwan  The pr LGTM now, thanks!
@pruan-rht  pls help to merge again if no objection, thanks!

File: lib/rules/web/admin_console/4.1/deployment.xyaml (hunk 1 above)
Comment (xiaocwan) [reply lib/rules/web/admin_console/4.1/deployment.xyaml]: PASS on Jenkins
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/44308/console

**Comment by xiaocwan** [comment]
@akostadinov Plz help to merge, thanks!

**Pull request merged**